data/battle_scripts_1.s

```diff
@@ -5567,11 +5565,13 @@ BattleScript_AlreadyAtFullHp::
 
 BattleScript_EffectFakeOut::
 	attackcanceler
-	jumpifnotfirstturn BattleScript_ButItFailedAtkStringPpReduce
+	jumpifnotfirstturn BattleScript_ButItFailedPpReduceAtkString
 	setmoveeffect MOVE_EFFECT_FLINCH
 	goto BattleScript_EffectHit
-
-BattleScript_ButItFailedAtkStringPpReduce::
+	
+BattleScript_ButItFailedPpReduceAtkStringRemoveItem::
+	removeitem BS_ATTACKER
+BattleScript_ButItFailedPpReduceAtkString::
```

Comment:
Why the change from ButItFailedAtkStringPpReduce? Seems unrcessary

Reply:
Because I felt that for the sake of consistency, each additional action would be added as a suffix. Forgot to PR that to pret.

Reply:
Then wouldn't it make more sense to change the other labels to read in the order the cmds happen?

Reply:
I'm thinking that the primary action should go first (`ButItFailed`)